hack/local-test.sh

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+# podman login quay.io
+# cp "${XDG_RUNTIME_DIR}/containers/auth.json" ~/.docker/config.json
+REGISTRY_AUTH_JSON="$HOME/.docker/config.json"
```

Comment:
```suggestion
REGISTRY_AUTH_JSON=${REGISTRY_AUTH_JSON:-"$HOME/.docker/config.json"}
```
I don't like storing credentials in `~/.docker/config.json` because they persist after restart (unlike `$XDG_RUNTIME_DIR`). Making it configurable would be good enough for me